### ISSUE TITLE
Test use myself datasets

### ISSUE THREAD

**Issue by leofanpico** [XingangPan/SCNN#53]
Hi ,Xingang
@XingangPan  你的网络很棒，我现在都已经跑通了，但在test阶段，我想测试自己的一些图片，所以我对list中的test.txt进行了修改，但是测试出现错误。我看了其他相关的issues，发现需要在test之前修改好自己的数据和list，或者需要删除gen文件夹下的文件，这两个方法我都试了试，总是出现如下错误，希望你能帮我解决这个问题，期望你的回答，thanks !

如下是错误信息：
           }
       ... -> output
  }
}
load file: /home/pico/project/SCNN_ROOT/datasets/CULane/list/test.txt	
/home/pico/torch/install/bin/luajit: /home/pico/project/SCNN_ROOT/datasets/laneTest-gen.lua:14: attempt to index local 'f' (a nil value)
stack traceback:
	/home/pico/project/SCNN_ROOT/datasets/laneTest-gen.lua:14: in function 'getPaths'
	/home/pico/project/SCNN_ROOT/datasets/laneTest-gen.lua:82: in function 'exec'
	./datasets/init.lua:28: in function 'create'
	./dataloader.lua:31: in function 'create'
	testLane.lua:25: in main chunk
	[C]: in function 'dofile'
	...pico/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405d50

**Comment by planckztd** [comment]
您好，请问为什么我跑他的测试代码会出现错误，请问这是不是环境出错了，还是因为一些路径的原因
data created	
data loaded	
data loaded	
./data/CULane/driver_100_30frame/05251517_0433.MP4/00000.jpg
./data/CULane/driver_100_30frame/05251517_0433.MP4/00030.jpg
1	
./data/CULane/driver_100_30frame/05251517_0433.MP4/00060.jpg
./data/CULane/driver_100_30frame/05251517_0433.MP4/00090.jpg
/home/kb457/torch/install/bin/luajit: /home/kb457/torch/install/share/lua/5.1/nn/Container.lua:67: 
In 2 module of nn.Sequential:
...torch/install/share/lua/5.1/cudnn/BatchNormalization.lua:69: bad argument #1 to 'resizeAs' (torch.CudaTensor expected, got userdata)
stack traceback:
	[C]: in function 'resizeAs'
	...torch/install/share/lua/5.1/cudnn/BatchNormalization.lua:69: in function <...torch/install/share/lua/5.1/cudnn/BatchNormalization.lua:63>
	[C]: in function 'xpcall'
	/home/kb457/torch/install/share/lua/5.1/nn/Container.lua:63: in function 'rethrowErrors'
	/home/kb457/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
	testLane.lua:71: in main chunk
	[C]: in function 'dofile'
	...b457/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405de0

WARNING: If you see a stack trace below, it doesn't point to the place where this error occurred. Please use only the one above.
stack traceback:
	[C]: in function 'error'
	/home/kb457/torch/install/share/lua/5.1/nn/Container.lua:67: in function 'rethrowErrors'
	/home/kb457/torch/install/share/lua/5.1/nn/Sequential.lua:44: in function 'forward'
	testLane.lua:71: in main chunk
	[C]: in function 'dofile'
	...b457/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x00405de0
